collections/_article/january-jams-2024.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "January Jams"
+excerpt: "Looking back at a month of GameJams, we curated a list of ten events that you might or might not have heard of before."
+categories: ["events"]
+author: Nathalie Galla
+image: /storage/blog/covers/godot-jam.webp
+image_caption_description: "Jam base image by <a href='https://www.freepik.com/free-photo/jam-isolated-white_1198739.htm#query=jam&position=39&from_view=search&track=sph&uuid=449f0bd4-ec2a-4a1b-aef3-d0ad383022a7'>luis_molinero</a> on Freepik"
```

Comment:
```suggestion
image_caption_description: "Jam base image by <a href='https://www.freepik.com/free-photo/jam-isolated-white_1198739.htm'>luis_molinero</a> on Freepik"
```
Removed search and tracking parameters.

Reply:
This one is a bit weird, because I get that when I copy paste the image credit, but then when trying to access it from a url without the tracking I wasn't able to load the page. So it seems to be somewhat required? If you can access it without the tracking parameters let me know and we can remove it 👍

Reply:
Ugh, you're right. I had initially clicked the long link, then I deleted the extra parameters and checked it still worked - which it did, but only in that same browser. If I try the shorter link in a fresh browser it doesn't work. I guess the site set a cookie or something on the first visit which allowed the second visit to work on the first browser.

So, yeah, it looks like the full URL is needed. Sorry for not testing more completely. I just have a hatred for messy URLs and thought I could fix it!

Reply:
Hate it too :( thanks for flagging it!